packages/techdocs-common/package.json

```diff
@@ -40,39 +40,9 @@
     "url": "https://github.com/backstage/backstage/issues"
   },
   "dependencies": {
-    "@azure/identity": "^2.0.1",
-    "@azure/storage-blob": "^12.5.0",
-    "@backstage/backend-common": "^0.12.0",
-    "@backstage/catalog-model": "^0.12.0",
-    "@backstage/config": "^0.1.15",
-    "@backstage/errors": "^0.2.2",
-    "@backstage/search-common": "^0.3.0",
-    "@backstage/integration": "^0.8.0",
-    "@google-cloud/storage": "^5.6.0",
-    "@trendyol-js/openstack-swift-sdk": "^0.0.5",
-    "@types/express": "^4.17.6",
-    "aws-sdk": "^2.840.0",
-    "express": "^4.17.1",
-    "fs-extra": "9.1.0",
-    "git-url-parse": "^11.6.0",
-    "js-yaml": "^4.0.0",
-    "json5": "^2.1.3",
-    "mime-types": "^2.1.27",
-    "mock-fs": "^5.1.0",
-    "p-limit": "^3.1.0",
-    "recursive-readdir": "^2.2.2",
-    "winston": "^3.2.1"
-  },
-  "devDependencies": {
-    "@backstage/cli": "^0.15.0",
-    "@types/fs-extra": "^9.0.5",
-    "@types/js-yaml": "^4.0.0",
-    "@types/mime-types": "^2.1.0",
-    "@types/mock-fs": "^4.13.0",
-    "@types/recursive-readdir": "^2.2.0",
-    "@types/supertest": "^2.0.8",
-    "supertest": "^6.1.3"
+    "@backstage/plugin-techdocs-node": "0.11.11"
```

Comment:
You made the other one a `^` range, so I'm assuming you want one of those two

Reply:
Ah, yeah.

Reply:
Eagle eyes!